transform-plugins/src/main/java/io/cdap/plugin/XMLToJSON.java

```diff
@@ -57,6 +59,14 @@ public XMLToJSON(Config config) {
     this.config = config;
   }
 
+  @Override
+  public void prepareRun(StageSubmitterContext context) throws Exception {
+    super.prepareRun(context);
+    context.record(
+      TransformLineageRecorderUtils.oneInToOneOut(config.inputField, config.outputField,
```

Comment:
In this transform, `config.inputField` is transformed to the json fields, which is one-to-many, all the other fields are identity one-to-one transform, your lineage should reflect this